sql/core/src/test/resources/sql-tests/results/subquery/in-subquery/nested-not-in.sql.out

```diff
@@ -0,0 +1,332 @@
+-- Automatically generated by SQLQueryTestSuite
+-- Number of queries: 21
+
+
+-- !query
+CREATE TEMPORARY VIEW EMP AS SELECT * FROM VALUES
+  (100, "emp 1", 10),
+  (200, "emp 2", NULL),
+  (300, "emp 3", 20),
+  (400, "emp 4", 30),
+  (500, "emp 5", NULL),
+  (600, "emp 6", 100),
+  (800, "emp 8", 70)
+AS EMP(id, emp_name, dept_id)
+-- !query schema
+struct<>
+-- !query output
+
+
+
+-- !query
+CREATE TEMPORARY VIEW DEPT AS SELECT * FROM VALUES
+  (10, "dept 1", "CA"),
+  (20, "dept 2", "NY"),
+  (30, "dept 3", "TX"),
+  (40, "dept 4 - unassigned", "OR"),
+  (50, "dept 5 - unassigned", "NJ"),
+  (70, "dept 7", "FL")
+AS DEPT(dept_id, dept_name, state)
+-- !query schema
+struct<>
+-- !query output
+
+
+
+-- !query
+CREATE TEMPORARY VIEW BONUS AS SELECT * FROM VALUES
+  ("emp 1", 10.00D),
+  ("emp 1", 20.00D),
+  ("emp 2", 300.00D),
+  ("emp 2", 100.00D),
+  ("emp 3", 300.00D),
+  ("emp 4", 100.00D),
+  ("emp 5", 1000.00D),
+  ("emp 6 - no dept", 500.00D)
+AS BONUS(emp_name, bonus_amt)
+-- !query schema
+struct<>
+-- !query output
+
+
+
+-- !query
+CREATE TEMPORARY VIEW ADDRESS AS SELECT * FROM VALUES
+  (100, "emp 1", "addr1"),
+  (200, null, "addr2"),
+  (null, "emp 3", "addr3"),
+  (null, null, "addr4"),
+  (600, "emp 6", "addr6"),
+  (800, "emp 8", "addr8")
+AS ADDRESS(id, emp_name, address)
+-- !query schema
+struct<>
+-- !query output
+
+
+
+-- !query
+CREATE TEMPORARY VIEW S1 AS SELECT * FROM VALUES
+  (null, null), (5, 5), (8, 8), (11, 11) AS s1(a, b)
+-- !query schema
+struct<>
+-- !query output
+
+
+
+-- !query
+CREATE TEMPORARY VIEW S2 AS SELECT * FROM VALUES
+  (7, 7), (8, 8), (11, 11), (null, null) AS s2(c, d)
+-- !query schema
+struct<>
+-- !query output
+
+
+
+-- !query
+SELECT id, 
+       dept_id 
+FROM   emp 
+WHERE  id = 600
+       OR id = 500 
+       OR dept_id NOT IN (SELECT dept_id 
+                          FROM   emp)
+-- !query schema
+struct<id:int,dept_id:int>
+-- !query output
+500	NULL
```

Comment:
I've checked that the output is the same with the PostgreSQL answers.

Reply:
does our ported pgsql test include "nested not in" queries?

Reply:
Ah, yea, we need to check that. I'll do later.

Reply:
I checked the ported tests don't have those queries. Actually, the original regression tests in PostgreSQL don't have, too.